### DISCUSSION TITLE
Project Fully Converted to C#

### DISCUSSION THREAD

**Discussion by Wiilf** [dylwedma11748/JTegraNX#12] [Show and tell]
I've converted the project to C# and I look to implement WinForms and all that. Source files - [https://filebin.net/txgmk422qoio2ktk/converted.zip](url)

There some occasional warnings noted by the conversion, but gives a good head start at this.

**Comment by dylwedma11748** [comment]
No doubt this would be interesting, but trying to fully convert JTegraNX to C# would be very difficult. For starters, JTegaNX relies heavily on JavaFX for it's UI and I'm 99.9% sure that JavaFX isn't an available package for C#, this would require a full redesign of the UI using a different toolkit.

And I like to think of code conversion tools as a concept generator that can give some inspiration on how it could look, but trying to compile a raw conversion from said tools almost never works (at least not for me), there would be a lot of re-writing to be done for this to happen.

But nevertheless, still an interesting concept and I may consider giving it a shot and seeing how it works.

**Comment by Wiilf** [comment]
Absolutely, however I've abandoned this project for integrating into our client. The codebase will need to see a full rewrite with basic reference from the conversion example. Unfortunately I have zero knowledge in Java, but I am proficient in 4 other languages. Do keep me updated if you wish to see a conversion in the near future.
